### DISCUSSION TITLE
Crashes

### DISCUSSION THREAD

**Discussion by GhostlyHoots** [rebtd7/FF13Fix#95] [General]
When I add the fix to FFXIII-2 in the right folder it just makes it crash on start up but works perfect for FFXIII. IDK what I'm doing wrong. I'm not using any mods too just trying to get the game to run smoothly. I'm running windows 11 pro with an I7 8086k and a RTX 3080.

[FF13Fix.log](https://github.com/rebtd7/FF13Fix/files/9441838/FF13Fix.log)

**Comment by GhostlyHoots** [comment]
Well figured out why it was crashing with the fix. Once you turn on compatibility mode on it crashes but once its off it doesn't. Just one thing tho if I keep it off the game has a super loud buzzing noise so thats not gonna work and the game isn't really that playable without the fix so idk what to do.

**Reply by GhostlyHoots** [reply]
Well figured it out. It was the razer surround sound that was enabled for my headset. Once I turned that off and disabled all enhancements the buzzing fully stopped without having to have compatibility mode on.

**Reply by Surihix** [reply]
The compatibility mode option will only work when using the 1.4.6 version. 
With the newer version of this fix, that option will cause the game to crash.

**Reply by GhostlyHoots** [reply]
Oh well don't need compatibility mode anymore anyway since found out what makes that bussing noise.
